Sources/Datadog/RUM/RUMMonitor/Scopes/RUMApplicationScope.swift

```diff
@@ -34,6 +48,24 @@ internal class RUMApplicationScope: RUMScope {
     let context: RUMContext
 
     func process(command: RUMCommand) -> Bool {
+        if let currentSession = sessionScope {
+            let shouldRefreshSession = currentSession.process(command: command)
```

Comment:
shouldn't this be `let should... = !currentSession.process(...)` as `process` returns `false` when `session` needs to be deallocated?